src/rpc/net.cpp

```diff
@@ -242,6 +242,8 @@ static RPCHelpMan getpeerinfo()
                 heights.push_back(height);
             }
             obj.pushKV("inflight", heights);
+            obj.pushKV("addr_processed", statestats.m_addr_processed);
+            obj.pushKV("addr_rate_limited", statestats.m_addr_rate_limited);
```

Comment:
is there a reason to omit this from the documentation?

Reply:
can use the descriptions in a5cc9e28f9e125a62168dcbe0b3484d098e56c1f if so inclined

Reply:
helps added in #22604.